OCD-5067-Certiification-Status-Chips

mattystank opened the pull request.

mattystank committed:
feat: enhance certification status display with icons and improve filter functionality

[#OCD-5067]

mattystank committed:
Merge branch 'staging' of https://github.com/chpladmin/chpl-website into OCD-5067

andlar commented on src/app/components/filter/filter-chips.jsx:
```diff
@@ -42,6 +44,13 @@ const useStyles = makeStyles(() => ({
+    backgroundColor: palette.white,
+  },
+  chipAvatar: {
+    backgroundColor: 'transparent!important',
```
```suggestion
    backgroundColor: 'transparent !important',
```

mattystank committed:
Update src/app/components/filter/filter-chips.jsx

Co-authored-by: Andrew Larned <andrew.larned@gmail.com>

mattystank committed:
Merge branch 'staging' into OCD-5067

mattystank committed:
fix: reorder certification status values for better clarity

[#OCD-5067]

mattystank committed:
Merge branch 'staging' into OCD-5067

kekey1 committed:
test[aqa]: CSS Selector specifies 'svg' element one level deep

A new 'svg' type of element was added in the HTML and was being
selected. The element being selected was no longer the "x" button to
remove a filter but instead was the new icon indicating certification
status. The more specific 'svg element one level deep' once again select
the correct element with the "x".

mattystank committed:
Merge pull request #7 from kekey1/OCD-5067-aqa

test[aqa]: CSS Selector specifies 'svg' element one level deep

mattystank committed:
Merge branch 'staging' into OCD-5067
